s3transfer/manager.py

```diff
@@ -27,26 +29,40 @@
 class TransferConfig(object):
     def __init__(self,
                  multipart_threshold=8 * MB,
-                 max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0,
+                 max_request_concurrency=10,
```

Comment:
Seems reasonable.

We'll probably need to loop back and work out good class names for the config classes to make sure no one confuses the similarly named transfer config classes we have.